frontend/src/app/shared/components/editable-toolbar-title/editable-toolbar-title.html

```diff
@@ -31,8 +31,8 @@
   <input-shim
     class="editable-toolbar-title--input editable-toolbar-title--input-shim toolbar--editable-toolbar">{{ selectedTitle || text.input_placeholder }}{{ selectedTitle ? '&nbsp;&nbsp;' : ''}}</input-shim>
 </div>
-<h2 *ngIf="!editable"
+<div *ngIf="!editable"
```

Comment:
This will affect all pages where the `editable-toolbar-title` component is used, but not all of them have a wrapping `h` tag around them..